tests/e2e/test_pvc_creation_deletion_scale.py

```diff
@@ -0,0 +1,219 @@
+"""
+Test to measure pvc scale creation & deletion time. Total pvc count would be 1500
+"""
+import logging
+import csv
+import random
+import pytest
+import threading
+
+from tests import helpers
+from ocs_ci.framework.testlib import scale, E2ETest, polarion_id
+from ocs_ci.ocs import constants
+
+log = logging.getLogger(__name__)
+
+
+@scale
+class TestPVCCreationDeletionScale(E2ETest):
+    """
+    Base class for PVC scale creation and deletion
+    """
+    @pytest.fixture()
+    def namespace(self, project_factory):
+        """
+        Create a new project
+        """
+        proj_obj = project_factory()
+        self.namespace = proj_obj.namespace
+
+    @pytest.mark.parametrize(
+        argnames=["access_mode", "interface"],
+        argvalues=[
+            pytest.param(
+                *[constants.ACCESS_MODE_RWO, constants.CEPHBLOCKPOOL],
+                marks=pytest.mark.polarion_id("OCS-1225")
+            ),
+            pytest.param(
+                *[constants.ACCESS_MODE_RWX, constants.CEPHBLOCKPOOL],
+                marks=pytest.mark.polarion_id("OCS-2010")
+            ),
+            pytest.param(
+                *[constants.ACCESS_MODE_RWX, constants.CEPHFS_INTERFACE],
+                marks=pytest.mark.polarion_id("OCS-2008")
+            ),
+        ]
+    )
+    @pytest.mark.usefixtures(namespace.__name__)
+    def test_1500_pvc_creation_deletion_scale(self, namespace, access_mode, interface):
```

Comment:
how about changing the test function name to:
test_multiple_pvc_creation_deletion_scale() so in the future we'll use this for other amounts with pytest params

Reply:
ack

Reply:
@ebenahar Updated code, request to provide approval.